docs/search-ui/guides-workplace-search.mdx

```diff
@@ -0,0 +1,156 @@
+---
+id: tutorials-workplace-search
+slug: /tutorials/workplace-search
+title: Workplace Search
+date: 2022-04-18
+tags: ["demo"]
+---
+
+This guide will walk you through getting up and running with a Search UI using [Elastic Workplace Search](https://www.elastic.co/workplace-search/) as a backend.
+
+Have problems with the setup? Let us know in the [feedback issue](https://github.com/elastic/search-ui/issues/707), and we'll try to help.
+
+## 🚨 Technical preview 🚨
+
+Workplace Search connector for Search UI is currently in technical preview status. It is not ready for production use.
+
+## Prerequisites
+
+Before continuing with this guide, make sure you:
+1) have a running Elastic deployment with Enterprise Search and 
+2) indexed some data in Workplace Search.
+
+The best way to get started with Elastic Workplace Search is to follow our [Getting Started guide](https://www.elastic.co/guide/en/workplace-search/current/workplace-search-getting-started.html).
+
+To index data into Workplace Search, read the [Content Sources Overview](https://www.elastic.co/guide/en/workplace-search/current/workplace-search-content-sources.html) and follow a guide for any content source available there. 
+
+## Creating a search page
+
+Have you indexed data into Workplace Search? Great! Let's create a search page in your application.
+
+We'll use React components in this tutorial, but you can use Search UI with any UI framework and even vanilla JavaScript. Read our <DocLink id="guides-philosophy" /> to learn more about using Search UI outside of React.
+
+First, let's install Search UI packages:
+
+```shell
+npm install @elastic/search-ui @elastic/react-search-ui @elastic/react-search-ui-views @elastic/search-ui-workplace-search-connector
```

Comment:
Just these two will do. The other dependencies are transitive.

```suggestion
npm install @elastic/react-search-ui @elastic/search-ui-workplace-search-connector
```

Reply:
Oh, awesome, good to know.